docs/workflow/requirements/linux-requirements.md

```diff
@@ -1,65 +1,100 @@
-Requirements to build dotnet/runtime on Linux
-======================
+# Requirements to build dotnet/runtime on Linux
 
-This guide will walk you through the requirements to build dotnet/runtime on Linux. Before building there is environment setup that needs to happen to pull in all the dependencies required by the build. There are two suggested ways to go about doing this. First you are able to use the Docker environments provided by https://github.com/dotnet/dotnet-buildtools-prereqs-docker, or you can set up the environment yourself. The documentation will go over both ways of building. Using Docker allows you to leverage our existing images which already have an environment set up.
+* [Docker](#docker)
+* [Environment](#environment)
+  * [Toolchain Setup](#toolchain-setup)
+    * [Additional Requirements for Cross-Building](#additional-requirements-for-cross-building)
+      * [Alpine ARM Cross-Building Requirements](#alpine-arm-cross-building-requirements)
+  * [Unsupported OSes](#unsupported-oses)
 
-General instructions for building are [here](../README.md).
-Instructions for building CoreCLR for Linux are [here](../building/coreclr/linux-instructions.md).
+This guide will walk you through the requirements to build _dotnet/runtime_ on Linux. Before building there is environment setup that needs to happen to pull in all the dependencies required by the build.
 
+There are two suggested ways to go about doing this. The first one is you are able to use the [provided Docker environments](https://github.com/dotnet/dotnet-buildtools-prereqs-docker) used in the official builds, or you can set up the environment yourself. The documentation will go over both ways. Using Docker allows you to leverage our existing images which already have an environment set up, while using your own environment grants you better flexibility on having other tools at hand you might need.
```

Comment:
> https://github.com/dotnet/dotnet-buildtools-prereqs-docker

This link does not really help you to setup the docker environment.

Imagine you are somebody going through this for the first time and click on this link. It is just going to confuse you what you are supposed to do with it.

This link should be *after* the list of the suggested docker images for building, just to say how these docker images were built.

Reply:
How is it confusing? It's just giving a brief overview of what the doc's going to be about, instead of the unorganized mess it used to be.

Reply:
The people will click on the first link appears to achieve the task. If you know Docker and see a link to "provided Docker environments", you are very likely to click on it without reading further. The problem is that clicking on this link will get you to a repo with soup of 100+ dockerfiles that are hard to navigate.

My suggestion is:
- Change "The first one is you are able to use the [provided Docker environments](https://github.com/dotnet/dotnet-buildtools-prereqs-docker) used in the official builds, or you can set up the environment yourself." to just "You can use the Docker images used by official builds, or you can set up the environment yourself."
- Add "These Docker images are built using Dockerfiles maintained in https://github.com/dotnet/dotnet-buildtools-prereqs-docker repo." below the table of the suggested docker images.

Reply:
That sounds reasonable. Noted. Will implement it in the next iteration.